### PR TITLE
refactor(TreeItem): remove focus-within css

### PR DESCRIPTION
**Pull request by fabriciomendonca**
As discussed with Pete, there shouldn't be a background change in the `focus-within` state

### CODE REVIEW

File: src/components/Tree/TreeItem/TreeItem.tsx [hunk 1]
```diff
@@ -307,6 +306,10 @@ export const TreeItem = memo(
             </li>
         );
     },
+    (prevProps, nextProps) => {
+        console.log(prevProps, nextProps);
+        return false;
+    },
```
Comment (anxobotana) [inline src/components/Tree/TreeItem/TreeItem.tsx:312]: do we need this?

Comment (fabriciomendonca) [reply src/components/Tree/TreeItem/TreeItem.tsx:312]: Oh no. Sorry. Will remove it. Good catch

Comment (fabriciomendonca) [reply src/components/Tree/TreeItem/TreeItem.tsx]: Done

**Review by anxobotana — APPROVED** [review]
LGTM

**Pull request merged**